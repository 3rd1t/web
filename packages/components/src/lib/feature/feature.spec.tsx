import React from "react"
import { render } from "@testing-library/react"

import Feature from "./feature"

describe("Feature", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <Feature
        icon={
          <svg fill="none" viewBox="0 0 24 24" stroke="currentColor">
            <path
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth={2}
              d="M3 10h18M7 15h1m4 0h1m-7 4h12a3 3 0 003-3V8a3 3 0 00-3-3H6a3 3 0 00-3 3v8a3 3 0 003 3z"
            />
          </svg>
        }
        title="Fair pricing"
        description="Start for free and if you want more, get the plan that suits you best. No hidden fees."
      />,
    )
    expect(baseElement).toBeTruthy()
  })
})
