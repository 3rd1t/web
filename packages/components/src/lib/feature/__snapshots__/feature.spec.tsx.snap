// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature should render successfully 1`] = `
<body>
  <div>
    <div
      class="flex items-start space-x-1 md:space-x-4"
    >
      <div
        class="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
      >
        <span
          class="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
        >
          <svg
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 10h18M7 15h1m4 0h1m-7 4h12a3 3 0 003-3V8a3 3 0 00-3-3H6a3 3 0 00-3 3v8a3 3 0 003 3z"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </span>
      </div>
      <div
        class="flex-grow pl-4 lg:pl-0 lg:px-6"
      >
        <h2
          class="text-lg font-medium leading-6 text-carbon-900"
        >
          Fair pricing
        </h2>
        <p
          class="mt-1 text-base leading-6 text-carbon-600"
        >
          Start for free and if you want more, get the plan that suits you best. No hidden fees.
        </p>
      </div>
    </div>
  </div>
</body>
`;
